tensorrt_llm/_torch/auto_deploy/transformations/library/rope.py

```diff
@@ -1,6 +1,53 @@
+"""
```

Comment:
Fantastic description! Thank you. This is going to be a super important reference moving forward